azurerm/resource_arm_policy_definition_test.go

```diff
@@ -85,14 +144,29 @@ func testCheckAzureRMPolicyDefinitionDestroy(s *terraform.State) error {
 	return nil
 }
 
-func testAzureRMPolicyDefinition_basic(ri int) string {
-	return fmt.Sprintf(`
+func testAzureRMPolicyDefinition_basic(ri int, managementGroup bool) string {
```

Comment:
rather than adding the extra logic inside of this, could we revert this function and add a new one which returns the policy definition inside of a management group?

Reply:
yes, thats not as DRY, but much clearer. done.